### PR TITLE
feat: Add tutorial on how to migrate Cloud Foundry apps to Kubernetes

### PR DESCRIPTION
**Pull request by Akash-Nayak**
Signed-off-by: Akash Nayak <akash19nayak@gmail.com>

### CODE REVIEW

File: docs/tutorials/cloud-foundry.md [hunk 1]
```diff
@@ -0,0 +1,730 @@
...
+
+1. Install Move2Kube.
+
+   ```console
+   $ MOVE2KUBE_TAG='v0.3.0-rc.0' bash <(curl https://raw.githubusercontent.com/konveyor/move2kube/main/scripts/install.sh)
+   ```
+
+2. Install dependencies.
+  * [Docker](https://www.docker.com/get-started)
+  * [operator-sdk](https://docs.openshift.com/container-platform/4.1/applications/operator_sdk/osdk-getting-started.html#osdk-installing-cli_osdk-getting-started)
```
Comment (ashokponkumar) [inline docs/tutorials/cloud-foundry.md:26]: We are no longer creating operators. So this dependency no longer holds

File: docs/tutorials/cloud-foundry.md [hunk 2]
```diff
@@ -0,0 +1,730 @@
...
+
+   ```console
+   $ cd move2kube-demos
+   ```
+   Let's see the structure inside the `./samples/cloud-foundry` directory.
+
+   ```console
+   move2kube-demos git:(main) $ tree samples/cloud-foundry
+   samples/cloud-foundry
+   ├── cfapps.yaml
```
Comment (ashokponkumar) [inline docs/tutorials/cloud-foundry.md:56]: The CF apps format has changed. Check with @seshapad on the new format and let's update the demos repo

**Review by ashokponkumar — CHANGES_REQUESTED** [review]
Minor suggestions

**Comment by ashokponkumar** [comment]
@HarikrishnanBalagopal is this the right location after the refactoring of the structure and the right title that fits with our new intent?

File: docs/tutorials/cloud-foundry.md (hunk 1 above)
Comment (Akash-Nayak) [reply docs/tutorials/cloud-foundry.md]: I have removed the operator-sdk from the dependencies and added Podman as an alternative to Docker.

File: docs/tutorials/cloud-foundry.md (hunk 2 above)
Comment (Akash-Nayak) [reply docs/tutorials/cloud-foundry.md:56]: okay sure

**Pull request merged**